Core/Entities/User.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+
+namespace Core.Entities
+{
+    public class User : BaseModel
+    {
+        public bool Verified { get; set; }
```

Comment:
Proponowałbym to nazwać `IsVerified` (jakiekolwiek inne boole również jakby pasowało)